Libraries/react-native/react-native.js

```diff
@@ -60,11 +65,18 @@ var ReactNative = Object.assign(Object.create(require('React')), {
   // Plugins
   DeviceEventEmitter: require('RCTDeviceEventEmitter'),
   NativeModules: require('NativeModules'),
+  ReactIOSViewAttributes: require('ReactIOSViewAttributes'),
+  createReactIOSNativeComponentClass: require('createReactIOSNativeComponentClass'),
```

Comment:
Good to know. I'll remove this these from the next revision to this diff for now.